ql/src/test/org/apache/hadoop/hive/ql/io/orc/TestInputOutputFormat.java

```diff
@@ -1837,6 +1836,26 @@ public void testProjectedColumnSize() throws Exception {
     assertEquals(376804, result.getProjectedColumnsUncompressedSize());
   }
 
+  @Test
+  public void testAcidProjectedColumnSize() throws Exception {
+    Path file = new Path(HiveTestUtils.getFileFromClasspath("bucket_00952_0"));
+    FileSystem fs = FileSystem.getLocal(conf).getRaw();
+    
+    OrcInputFormat.Context context = new OrcInputFormat.Context(conf);
+    OrcInputFormat.SplitInfo splitInfo = new OrcInputFormat.SplitInfo(
+        context, fs, fs.getFileStatus(file), null, null, true, new ArrayList<>(), true, null, null);
+    OrcInputFormat.SplitGenerator splitter = new OrcInputFormat.SplitGenerator(splitInfo, null, true, true);
+    
+    List<OrcSplit> results = splitter.call();
+    assertEquals(1, results.size());
+    
+    OrcSplit result = results.get(0);
+    assertEquals(3, result.getStart());
+    assertEquals(900, result.getLength());
+    assertEquals(1, results.size());
+    assertEquals(1246255309, results.get(0).getProjectedColumnsUncompressedSize());
```

Comment:
nit: reuse result instead of results.get(0)

Reply:
removed result